doc/classes/PhysicsServer2D.xml

```diff
@@ -369,7 +370,8 @@
 		<method name="body_create">
 			<return type="RID" />
 			<description>
-				Creates a 2D body object in the physics server, and returns the [RID] that identifies it. Use [method body_add_shape] to add shapes to it, use [method body_set_state] to set its transform, and use [method body_set_space] to add the body to a space.
+				Creates a 2D body object in the physics server, and returns the [RID] that identifies it. The default settings for the created body include a collision layer and mask set to [code]1[/code], and [enum BodyMode] is set to [code]BODY_MODE_RIGID[/code].
```

Comment:
I'd restore this, it's outside the scope of this PR and should be reviewed separately, let's not drift here